en/file_formats/plan.md

```diff
@@ -98,40 +148,246 @@ A complex item is a higher level encapsulation of multiple MISSION_ITEMS treated
     ...
 },
 ```
-Complex items have two additional values associated with them:
+
+Complex items have these values associated with them:
 
 Key | Description
-----|------------
-complexItemType | Specifies the type of complex item. QGroundControl currently supports the following types: [survey](../file_formats/survey.md), fwLandingPattern
-version | Specifies the version for this complex item.
+--- | ---
+`TransectStyleComplexItem` | ?
```

Comment:
What is this? It seems to be a group for a particular type complex item, but then you ALSO have a type (survey). 
What are the other types of item?

Reply:
Survey and CorridorScan share a common base of information/code which is TransectStyleComplexItem.